### PR TITLE
Apply glossary builder to replace vocabulary table

### PR DESCRIPTION
**Pull request by terrymyc**
This PR applies the glossary builder to replace the vocabulary table. The term can be referred to anywhere within the documentation by:

> :term:\`<TERM_NAME>\`

Read more about [linking a term to its glossary entry](https://sublime-and-sphinx-guide.readthedocs.io/en/latest/glossary.html#link-a-term-to-its-a-glossary-entry)

The main text isn't changed too much.

image

### CODE REVIEW

File: docs/source/intro/glossary.rst [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+  ASReview CLI
+    ASReview CLI is the command line interface that is developed for advanced
+    options or for running simulation studies.
+
+  ASReview LAB
+    ASReview LAB is the graphical user interface.
+
+  ASReview project
+    ASReview project is an encompassing term for all work that is done by the
+    ASReview team members and ASReview :term:`contributors<contributor>`.
```
Comment (Rensvandeschoot) [inline docs/source/intro/glossary.rst:29]:
```suggestion
    [ASReview team members](https://asreview.nl/about) and ASReview :term:`contributors<contributor>`.
```

File: docs/source/intro/glossary.rst [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+ASReview LAB makes use of standardized terminology for all communication
+regarding ASReview LAB and its underlying technology.
+
+.. glossary::
+
+  Active learning model
+    Active learning model is used to indicate how to select the next
+    :term:`record` to be reviewed by the :term:`user`. The model consists of
+    several elements: a feature extraction technique, a classifier, a balance
+    and a query strategy.
```
Comment (Rensvandeschoot) [inline docs/source/intro/glossary.rst:13]:
```suggestion
    Active learning model is used to indicate the combination of four 
    elements: a feature extraction technique, a classifier, a balance
    and a query strategy.
```

Comment (Rensvandeschoot) [reply docs/source/intro/glossary.rst:13]: I think the original first sentence refers only the query strategy.

File: docs/source/intro/glossary.rst [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+    :term:`ASReview` mascot. It is used for storytelling and to increase
+    explainability.
+
+  Export
+    Export is the action of exporting a :term:`dataset` or a :term:`project`
+    from :term:`ASReview LAB`.
+
+  Extension
+    Extension is the additional element to the :term:`ASReview LAB`, such as
+    the ASReview visualisation extension, or the ASReview CORD-19 extension.
```
Comment (Rensvandeschoot) [inline docs/source/intro/glossary.rst:54]: Do we want a link to the documentation about extensions?

File: docs/source/intro/glossary.rst [hunk 4]
```diff
@@ -0,0 +1,149 @@
...
+    ASReview stands for *Active learning for Systematic Reviews* or
+    *AI-assisted Systematic Reviews*, depending on context. Avoid this
+    explanation, only use as tagline.
+
+  ASReview CLI
+    ASReview CLI is the command line interface that is developed for advanced
+    options or for running simulation studies.
+
+  ASReview LAB
+    ASReview LAB is the graphical user interface.
```
Comment (Rensvandeschoot) [inline docs/source/intro/glossary.rst:25]:
```suggestion
    ASReview LAB is the graphical user interface of the open-source research software 
    to explore the future of AI in systematic reviews.
```

**Review by Rensvandeschoot — CHANGES_REQUESTED** [review]

**Comment by Rensvandeschoot** [comment]
This glossary is much better compared to the old table!!!

File: docs/source/intro/glossary.rst (hunk 3 above)
Comment (terrymyc) [reply docs/source/intro/glossary.rst]: Added a link to the visualization. Maybe not for CORD-19 as it is deprecated.

**Review by Rensvandeschoot — APPROVED** [review]

**Pull request merged**